data/mustafaansarii.json

```diff
@@ -0,0 +1,85 @@
+{
+  "name": "Mustafa Ansari",
+  "type": "personal",
+  "bio": "Open Source Dev | Computer Science Student | software developer | DSA | A geek | investor",
+
+  "tags": [
+    "Open Source",
+    "Java",
+    "HTML",
+    "CSS",
+    "Python",
+    "DSA",
+    "Blogs"
+  ],
+  "socials": [
+    { "icon": "FaTwitter", "url": "https://twitter.com/mustafa_nds" },
+    { "icon": "FaGithub", "url": "https://github.com/mustafaansarii" },
+    { "icon": "FaYoutube", "url": "https://www.youtube.com/@mustafasmartX" },
+    { "icon": "FaLinkedin", "url": "https://www.linkedin.com/in/mustafa-ansari-75059a237/" },
+    { "icon": "FaInstagram", "url": "https://www.instagram.com/mustafa.nds/" }
+  ],
+  "testimonials": [
+    "Rahul kr",
+    "Sachin singh",
+    "nikhil raj",
+    "Ankit kr"
+    
+  ],
```

Comment:
This is not how testimonials work, ask your testimonials to refer [these docs](https://linkfree.eddiehub.io/docs/how-to-guides/testimonials) and follow the required steps, then you can add their GitHub usernames here.

For now, it's better to remove this from your `json`. 

```suggestion
```